templates/partials/summary/collapsible-summary.html

```diff
@@ -36,7 +36,8 @@
                             _("No answer provided"),
                             _("Change"),
                             _("Change your answer for:"),
-                            content.summary.calculated_question
+                            content.summary.calculated_question,
+                            None
```

Comment:
If Jinja supports keyword argument, we should use it, as this has no meaning with it. Without checking the code, we don't know what `None` means.

Reply:
Seems like jinja supports keyword args. Added in both places where config method is called.